loadgen/generic_executor_test.go

```diff
@@ -12,27 +12,56 @@ import (
 	"go.uber.org/zap"
 )
 
-func TestIterationsAndDuration(t *testing.T) {
-	info := ScenarioInfo{MetricsHandler: client.MetricsNopHandler}
-
-	var err error
-	{
-		// only iterations
-		executor := &GenericExecutor{DefaultConfiguration: RunConfiguration{Iterations: 3}}
-		_, err = executor.newRun(info)
-		require.NoError(t, err)
-	}
-	{
-		// only duration
-		executor := &GenericExecutor{DefaultConfiguration: RunConfiguration{Duration: time.Hour}}
-		_, err = executor.newRun(info)
-		require.NoError(t, err)
+func TestScenarioConfigValidation(t *testing.T) {
+	tests := []struct {
+		name  string
+		info  ScenarioInfo
+		error string
+	}{
+		{
+			name:  "default",
+			info:  ScenarioInfo{Configuration: RunConfiguration{}},
+			error: "",
```

Comment:
Left the default value to signal to the reader that this is expected to be valid.